.github/workflows/nightly.yml

```
@@ -1,176 +1,214 @@
-            -v \
-            -n auto
-          TEST_END=$(date +%s)
-          TEST_DURATION=$((TEST_END - TEST_START))
+            -v
 
-          # Generate performance report with actual timing
-          echo "## Performance Report" > performance-report.md
-          echo "Generated: $(date)" >> performance-report.md
+      - name: Generate test report
+        if: always()
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2129:style:5:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)